backend/app/api/routes/chat.py

```
@@ -0,0 +1,126 @@
+            if request.session_id:
+                session_store.add_message(request.session_id, user_message)
+                assistant_msg = Message(role="assistant", content=response_text)
+                session_store.add_message(request.session_id, assistant_msg)
+
+            return {
+                "message": response_text,
+                "model_id": request.model_id,
+                "session_id": request.session_id
+            }
+
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)